common/views/components/OpeningHours/OpeningHours.js

```diff
@@ -39,7 +39,7 @@ class OpeningHours extends Component<Props, State> {
       <Fragment>
         {groupedVenues && !groupedVenues[Object.keys(groupedVenues)[0]].hours &&
           <p className={spacing({s: 2}, {margin: ['bottom']})}>
-            <a className={font({s: 'HNL6', m: 'HNL5'})} href='https://wellcomecollection.org/info/opening-times'>
+            <a className={font({s: 'HNL6', m: 'HNL5'})} href='https://wellcomecollection.or/opening-times'>
```

Comment:
typo .or/

Reply:
and will need to add ^/opening-times to router

Reply:
@gestchild Ah yes - fixed.